pkg/hintrunner/zero/zerohint_keccak.go

```diff
@@ -479,9 +479,68 @@ func newBlockPermutationHint(keccakPtr hinter.ResOperander) hinter.Hinter {
 
 func createBlockPermutationHinter(resolver hintReferenceResolver) (hinter.Hinter, error) {
 	keccakPtr, err := resolver.GetResOperander("keccak_ptr")
-	if keccakPtr != nil {
+	if err != nil {
 		return nil, err
 	}
 
 	return newBlockPermutationHint(keccakPtr), nil
 }
+
+func newCompareBytesInWordNondetHint(nBytes hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "CompareBytesInWordNondetHint(",
+		Op: func(vm *VM.VirtualMachine, _ *hinter.HintRunnerContext) error {
+			//> memory[ap] = to_felt_or_relocatable(ids.n_bytes < ids.BYTES_IN_WORD)
```

Comment:
I know that in whitelisted json file, this is the hint written, but if you check the flow in keccak stuff, the hint is : 

```
ids.n_bytes < ids.BYTES_IN_WORD
```
see : 

https://github.com/starkware-libs/cairo-lang/blob/efa9648f57568aad8f8a13fbf027d2de7c63c2c0/src/starkware/cairo/common/cairo_keccak/keccak.cairo#L211